src/utils/helper.ts

```diff
@@ -33,7 +33,7 @@ export function isComponentInstance(obj: any) {
 
 export function createSlotProxy(vm: ComponentInstance, slotName: string) {
   return (...args: any) => {
-    if (!vm.$scopedSlots[slotName]) {
+    if (__DEV__ && !vm.$scopedSlots[slotName]) {
```

Comment:
This changes the behaviour, should return silently on prod

Reply:
fixed